.github/workflows/build-and-deploy.yml

```diff
@@ -42,8 +35,6 @@ jobs:
           filters: |
             ci:
               - '.github/workflows/build-and-deploy.yml'
-            deployment:
-              - 'deployments/**'
```

Comment:
The deployment process is outsourced to [ec2-deployment](https://github.com/Knoblauchpilze/ec2-deployment).